### PR TITLE
feat: set poppins font family

### PR DESCRIPTION
**Pull request by juaoose**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -271,6 +271,7 @@ class _AppState extends State<App> {
       child: MaterialApp(
         title: 'RealtimeChat',
         theme: ThemeData(
+          fontFamily: 'Poppins',
```
[the remark this answers is not in the document]
Comment (juaoose) [reply lib/main.dart:274]: Should I not apply it globally then?

[the remark this answers is not in the document]
Comment (kevmo314) [reply lib/main.dart:274]: We should apply it globally but also override the chat panel font

**Comment by juaoose** [comment]
I am not sure how to retrieve the "default" font after it has been set on the theme, so I just retrieved it from the material themes... is that ok?

**Comment by kevmo314** [comment]
> I am not sure how to retrieve the "default" font after it has been set on the theme, so I just retrieved it from the material themes... is that ok?

That's ok :)

**Review by kevmo314 — APPROVED** [review]

**Pull request merged**